mix.exs

```diff
@@ -18,12 +18,12 @@ defmodule PhoenixPubsubRedis.Mixfile do
   end
 
   def application do
-    [applications: [:logger, :redo, :poolboy]]
+    [applications: [:logger, :poolboy]]
```

Comment:
My understanding is that redix does not run as an OTP application, but just as part of the supervision tree?